trie/sync.go

```diff
@@ -292,8 +295,11 @@ func (s *Sync) children(req *request, object node) ([]*request, error) {
 			if _, ok := s.membatch.batch[hash]; ok {
 				continue
 			}
-			if ok, _ := s.database.Has(node); ok {
-				continue
+			if s.bloom.Contains(node) {
```

Comment:
There are two more location where (I think) the bloom should be used. Around line 101 in `AddSubTrie` and line 139 in `AddRawEntry`

Reply:
Ah yeah, I was looking a those a while back too. Indeed could help a bit.